src/components/our_projects.rs

```diff
@@ -0,0 +1,167 @@
+use yew::{html, Component, ComponentLink, Html, ShouldRender};
+use yew::services::fetch::{FetchService, FetchTask, Request, Response};
+use super::{project_card::{ProjectCard}};
+use yew::services::ConsoleService;
+use yew::format::Nothing;
+use anyhow::Error;
+use serde::Deserialize;
+
+const GITHUB_REPOS_URL: &str = "https://api.github.com/orgs/rust-lang-ar/repos";
+
+  
+pub struct OurProjects {
+    is_fetching: bool,
+    fetch_failed: bool,
+    _fetch_task: Option<FetchTask>,
```

Comment:
Why do we have underscore-prefixed field names? If we have field access keywords for that?